src/views/Asset.vue

```diff
@@ -284,16 +284,18 @@
                   :placeholder="'0.00 ' + (tokenSelected ? formAssetName(assetName, asset[tokenSelected]) + ' ' : '') + 'Tokens'"
                 />
                 <input
-                  v-if="tokenSelected && navAct != 'redeem' && navAct !== 'lptrade'"
+                  v-if="tokenSelected && navAct != 'redeem' && navAct !== 'lptrade' && withdrawType != 'existing'"
                   id
                   class="numeric setvalue"
                   type="number"
                   name
                   v-model="collatAmt"
                   v-on:keyup="collatHandler"
                   :placeholder="'0.00 ' + asset[tokenSelected].collateral + (navAct === 'mint' ? ' Collateral' : '')"
-                  :disabled="navAct == 'withdraw' && withdrawType == 'existing'"
                 />
+                <label class="withdrawLabel" v-if="navAct == 'withdraw' && withdrawType == 'existing'">
+                  <b>{{ '0.00 ' + asset[tokenSelected].collateral + (navAct === 'mint' ? ' Collateral' : '') }}</b>
```

Comment:
this should be dynamic, and to look like this example: **Withdraw XX USDC**
- and this part isnt needed `(navAct === 'mint' ? ' Collateral' : '')`
```suggestion
                  <b>{{ (asset[tokenSelected].collateral == "WETH" ? numeral(collatAmt, "0.0000a") : numeral(collatAmt, "0.00a")) + ' ' + asset[tokenSelected].collateral }}</b>
```